securedrop_client/gui/login_dialog.py

```diff
@@ -35,12 +35,8 @@
 )
 
 from securedrop_client import __version__ as sd_version
-from securedrop_client.gui.widgets import (
-    LoginErrorBar,
-    LoginOfflineLink,
-    PasswordEdit,
-    SignInButton,
-)
+from securedrop_client.gui.auth import SignInButton
```

Comment:
This import is an indicator that `LoginDialog` belongs in `auth`. (Hold that thought.)

Why?

*Generally*: The `securedrop_client.gui` package holds all the generic building blocks for the GUI. If this class needs knowledge about an internal package, it's because it belong to it.

**Rationale**: It is fine for any given module to require other modules from the package it belong to, but it wouldn't be fine to reach into the internals of a package you depend on.

**Specifics in this case**: in this case -and until new notice- the `LoginDialog` belongs to `gui`, so importing `gui` is fine, but `gui.auth` wouldn't. Since we are importing `gui.auth`, it is an indication that the class actually belongs to that package.) This will be a recurrent pattern as we extract `auth`-related widgets from `widgets.py`, it's useful to keep it in mind to see how ultimately `LoginDialog` finds its place in `auth` at the same time the circular dependencies get resolved.